<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>Processor App</string>
  </property>

  <property name="styleSheet">
   <string notr="true">

    QMainWindow {
        background-color: #1e1e2f;
    }

    QWidget {
        background-color: #1e1e2f;
        color: white;
        font-family: Segoe UI;
        font-size: 14px;
    }

    QLineEdit {
        background-color: #2d2d44;
        border-radius: 10px;
        padding: 8px;
        border: 1px solid #444;
        color: white;
    }

    QLineEdit:focus {
        border: 1px solid #00c6ff;
    }

    QPushButton {
        border-radius: 10px;
        padding: 8px;
        font-weight: 600;
        color: white;
        border: none;
        background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:0,
            stop:0 #00c6ff,
            stop:1 #0072ff
        );
    }

    QPushButton:hover {
        background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:0,
            stop:0 #00e5ff,
            stop:1 #2979ff
        );
    }

    QPushButton:pressed {
        background-color: #005ecb;
    }

    QCheckBox {
        spacing: 8px;
        font-size: 15px;
    }

    QCheckBox::indicator {
        width: 18px;
        height: 18px;
        border-radius: 5px;
        border: 2px solid #666;
        background-color: #2d2d44;
    }

    QCheckBox::indicator:checked {
        background-color: #00c853;
        border: 2px solid #00e676;
    }

   </string>
  </property>

  <widget class="QWidget" name="centralwidget">

   <!-- Input Folder -->
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>450</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Folder</string>
    </property>
   </widget>

   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <!-- Output Folder -->
   <widget class="QLineEdit" name="filename2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>450</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Output Folder</string>
    </property>
   </widget>

   <widget class="QPushButton" name="browse2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <!-- Subfolder checkbox -->
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Include Sub Folder</string>
    </property>
   </widget>

   <!-- OK Button -->
   <widget class="QPushButton" name="okbtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>180</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Process</string>
    </property>
   </widget>

  </widget>

  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>

 </widget>
 <resources/>
 <connections/>
</ui>